www/js/survey/enketo-survey-service.js

```diff
@@ -103,23 +103,19 @@ angular.module('emission.enketo-survey.service', [
     return answer;
   }
 
+
   function getAllSurveyAnswers(key = 'manual/confirm_survey', opts = {}) {
     const _opts_populateLabels = opts.populateLabels || false;
 
     const tq = $window.cordova.plugins.BEMUserCache.getAllTimeQuery();
     return UnifiedDataLoader.getUnifiedMessagesForInterval(key, tq)
     .then(function(answers){
       if (!_opts_populateLabels) return answers;
-
       const xmlParser = new $window.DOMParser();
       if (key === 'manual/confirm_survey') {
         return answers.map(function(answer){
           return populateLabels(answer, xmlParser);
         });
-      }
-
-      return answers;
```

Comment:
@asiripanich you need to fix this too. Basically, there should be no changes in `enketo-survey-service.js`.